code-challenges.js

```diff
@@ -3,6 +3,39 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// Function that returns an array with 10 elements.
+// Need a loop with a length of 10
+// Need 2 variables to sum numbers
+// Need 1 variable to store summed up number's answer
+// Need an empty array to push answers to
+// 0  + 1  = index1
+// n1 + n2 = index1
+// n2 + n3 = index2
+
+// PLEASE SHOW ME A MORE EFFICIENT WAY TO WRITE THIS IF POSSIBLE!
+
+const fibonacci = () => {
+  // n1 and n2 represents the starting number of the fibonacci sequence
+  // n1 and n2's value will change within a loop
+  let n1 = 0;
+  let n2 = 1;
+  // result array will store the sum of each n1/n2 answers
+  let result = [];
+  // for loop will execute 10 times
+  for (let i = 0; i < 10; i++) {
+    // answer's value is summed up
+    let answer = n1 + n2;
+    // answer is pushed to the result array
+    result.push(answer);
+    // Numbers are updated for next sequence
+    n2 = n1;
+    n1 = answer;
+  }
+  return result;
+}
+
```

Comment:
good job adding very clear comments on this code. without the comments, it might be difficult to see what exactly you're trying to do with this code. for a more efficient solution see: recursion. that can be a fun bit of extra credit for you :)